kubeportal/templates/portal_welcome.html

```diff
@@ -21,9 +21,34 @@
           Sorry, you have no access to the cluster.
         </p>
         {% if user.has_access_requested %}
```

Comment:
I only see a difference in the text, the form is the same. Please optimize that.